core/src/main/java/com/yandex/yoctodb/mutable/AbstractDocumentBuilder.java

```diff
@@ -85,14 +85,23 @@ public DocumentBuilder withField(
     @Override
     public DocumentBuilder withField(
             @NotNull
-            final
-            String name,
+            final String name,
             final long value,
             @NotNull
             final IndexOption index) {
         return withField(name, UnsignedByteArrays.from(value), index, LengthOption.FIXED);
     }
 
+    public DocumentBuilder withField(
+            @NotNull
+            final String name,
+            final char value,
+            @NotNull
+            final IndexOption index
+    ) {
```

Comment:
It will look more consistent to join with the previous line.

Reply:
Fixed